### PR TITLE
Added docstring ops/image/resize

### PR DESCRIPTION
**Pull request by Frightera**
PR adds the docstring for `keras_core.ops.image.resize`

### CODE REVIEW

File: keras_core/ops/image.py [hunk 1]
```diff
@@ -55,7 +55,35 @@ def compute_output_spec(self, image):
...
+        data_format: Format of the image tensor. Defaults to 'channels_last'.
+
+    Returns:
+        Resized image or batch of images.
+
+    Examples:
+
+    >>> x = np.random.random((2, 4, 4, 3)) # batch of 2 RGB images
+    >>> y = keras_core.ops.image.resize(x, (2, 2))
+    >>> y.shape
```
Comment (fchollet) [inline keras_core/ops/image.py:75]: Print shape (same below)

File: keras_core/ops/image.py [hunk 2]
```diff
@@ -55,7 +55,35 @@ def compute_output_spec(self, image):
...
+
+    >>> x = np.random.random((4, 4, 3)) # single RGB image
+    >>> y = keras_core.ops.image.resize(x, (2, 2))
+    >>> y.shape
+
+    >>> x = np.random.random((2, 3, 4, 4)) # batch of 2 RGB images
+    >>> y = keras_core.ops.image.resize(x, (2, 2),
+    ...     data_format='channels_first')
```
Comment (fchollet) [inline keras_core/ops/image.py:83]: Prefer using `"`

File: keras_core/ops/image.py [hunk 3]
```diff
@@ -55,7 +55,35 @@ def compute_output_spec(self, image):
...
-    # TODO: add docstring
+    """Resize images to size using the specified method.
+
+    Args:
+        image: Input image or batch of images. Must be 3D or 4D.
+        size: Size of output image in (height, width) format.
+        method: Interpolation method. Defaults to 'bilinear'.
+        antialias: Whether to use an antialiasing filter when downsampling an
+            image.
+        data_format: Format of the image tensor. Defaults to 'channels_last'.
```
Comment (fchollet) [inline keras_core/ops/image.py:66]: Copy the description from other places that uses this arg

File: keras_core/ops/image.py [hunk 4]
```diff
@@ -55,7 +55,35 @@ def compute_output_spec(self, image):
 def resize(
     image, size, method="bilinear", antialias=False, data_format="channels_last"
 ):
-    # TODO: add docstring
+    """Resize images to size using the specified method.
+
+    Args:
+        image: Input image or batch of images. Must be 3D or 4D.
+        size: Size of output image in (height, width) format.
+        method: Interpolation method. Defaults to 'bilinear'.
```
Comment (fchollet) [inline keras_core/ops/image.py:63]: List available methods. They're code keywords, so use both backticks and quotes, e.g. `"bilinear"`

File: keras_core/ops/image.py [hunk 5]
```diff
@@ -55,7 +55,35 @@ def compute_output_spec(self, image):
 def resize(
     image, size, method="bilinear", antialias=False, data_format="channels_last"
 ):
-    # TODO: add docstring
+    """Resize images to size using the specified method.
+
+    Args:
+        image: Input image or batch of images. Must be 3D or 4D.
+        size: Size of output image in (height, width) format.
```
Comment (fchollet) [inline keras_core/ops/image.py:62]: Backticks around tuple

**Review by fchollet — COMMENTED** [review]
Thanks for the PR!

File: keras_core/ops/image.py (hunk 1 above)
Comment (Frightera) [reply keras_core/ops/image.py:75]: That depends on the backend, if it is TensorFlow it'll output `TensorShape` etc. Guess it is best to remove `y.shape` then?

Comment (Frightera) [reply keras_core/ops/image.py:75]: Is it OK to put symbolic outputs here, like: `(2, 2, 2, 3)` etc?

**Comment by Frightera** [comment]
Thank you for your comments @fchollet, should be good to go now.

**Comment by Frightera** [comment]
Hi @fchollet, any update on this? Looks like I can't re-run the checks 😢

File: keras_core/ops/image.py (hunk 1 above)
Comment (fchollet) [reply keras_core/ops/image.py:83]: Yes, it should just be indicative.

**Review by fchollet — APPROVED** [review]
LGTM, thanks!

**Pull request merged**